packages/fhir-router/src/fhirrouter.ts

```diff
@@ -5,6 +5,7 @@ import { processBatch } from './batch';
 import { graphqlHandler } from './graphql';
 import { FhirRepository } from './repo';
 import { HttpMethod, Router } from './urlrouter';
+import EventEmitter from 'events';
```

Comment:
I think this is a Node.js-only import?  `fhir-router` is used in Storybook, which will probably choke on this.

The good news is that you should be able to use `EventTarget` from `@medplum/core`

Then replace `emit` with `dispatchEvent` and `on` with `addEventListener`